src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer stopped its work");
+    }
+}
```

Comment:
do not forget about @Override. fix everywhere
```suggestion
public class Bulldozer extends Machine {
    @Override
    public void doWork() {
        System.out.println("Bulldozer started its work");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer stopped its work");
    }
}
```